lib/handlers/api.js

```diff
@@ -12,6 +12,8 @@ var eventMiddleware = require('../middleware/events');
 var appdataRouter = require('./app/data.js');
 var deleteEnvironmentData = require('../services/environment/deleteEnvironmentData.js');
 var logger = require('../util/logger.js').getLogger();
+var cleanup = require('../handlers/cleanup/tmp_cleanup');
```

Comment:
The code base in fh-mbaas is largely still ES5 and so i decided to keep it that way. The alternative would be to move everything to const/let here.